.github/workflows/code_checks.yml

```diff
@@ -88,6 +108,27 @@ jobs:
             docker compose -f docker-compose.test.yml run web bash \
               -c "CI=true RAILS_ENV=test DISABLE_BOOTSNAP=true bundle exec parallel_test -n 13 -e 'bin/rails db:reset'"
 
+      - name: Remove Review label
```

Comment:
Added lint failure before tests begin. Labels will be removed after check passes.